### ISSUE TITLE
Support Tailwind CSS 3

### ISSUE THREAD

**Issue by diamondobama** [rogden/tailwind-config-viewer#51]
Tailwind CSS 3 was officially released yesterday but I couldn't install it on Nuxt because of below:

    "peerDependencies": {
      "tailwindcss": "1 || 2 || 2.0.1-compat"
    },

When should we expect an update for TCV to support version 3 of Tailwind?

**Comment by rogden** [comment]
Thanks for the issue. I'll issue a fix this weekend. In the meantime you can disable the tailwind viewer in Nuxt and run: `npx tailwind-config-viewer` in the directory that contains your tailwind.config.js file.

**Comment by michaelpumo** [comment]
I also get this issue. I need to run `npm i --force` in order to get dependencies installed (not ideal!).

Once I do though, config viewer appears to work fine with Tailwind 3...so I guess it's just because of config/dependency settings in this package.

```
➜  prosperly git:(main) ✗ npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: prosperly@1.0.0
npm ERR! Found: tailwindcss@3.0.7
npm ERR! node_modules/tailwindcss
npm ERR!   dev tailwindcss@"^3.0.7" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer tailwindcss@"1 || 2 || 2.0.1-compat" from tailwind-config-viewer@1.6.2
npm ERR! node_modules/tailwind-config-viewer
npm ERR!   dev tailwind-config-viewer@"^1.6.2" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by rogden** [comment]
Fixed in 1.6.3
